### PR TITLE
9.1/copyright footer dates

### PR DESCRIPTION
**Pull request by trasher**

### CODE REVIEW

File: config/define.php [hunk 1]
```diff
@@ -37,6 +37,7 @@
 
 // Current version of GLPI
 define("GLPI_VERSION","9.1");
+define("GLPI_YEAR","2016");
```
Comment (remicollet) [inline config/define.php]: Plzase, single quote

File: inc/html.class.php [hunk 2]
```diff
@@ -5797,5 +5772,20 @@ static function jsAlertCallback( $msg, $title, $okCallback=null) {
          }).text('".Toolbox::addslashes_deep($msg)."');
          " ;
    }
+
+   /**
+    * Get copyright message as HTML (used in footers)
+    *
```
Comment (remicollet) [inline inc/html.class.php]: Missing the "@since 9.1" in doc block

File: inc/html.class.php [hunk 3]
```diff
@@ -5797,5 +5772,20 @@ static function jsAlertCallback( $msg, $title, $okCallback=null) {
...
+    * @return text
+    */
+   static function getCopyrightMessage() {
+      $message = "<a href=\"http://glpi-project.org/\" title=\"Powered By Teclib\">";
+      $message .= "<span class=\"copyright\">GLPI " .
```
Comment (remicollet) [inline inc/html.class.php]: I'm not sure about the need for the span which was present only in 2 usage on 4.

File: config/define.php (hunk 1 above)
Comment (trasher) [inline config/define.php]: Heu.. All other defines have double quotes in that file... :/

File: inc/html.class.php (hunk 2 above)
Comment (trasher) [inline inc/html.class.php]: Will add, tanks

File: inc/html.class.php (hunk 3 above)
Comment (trasher) [inline inc/html.class.php]: The .copyright class do exists in the css file, but we can set it on the "a" element and remove the span.

**Comment by remicollet** [comment]
Perhaps a simple test case ?
- return value contains GLPI_VERSION and GLPI_YEAR

Additionally a test which checks that GLPI_YEAR == date('Y') so we'll not forget to change it ;)

**Comment by ddurieux** [comment]
Very good idea for the test ;)

File: css/styles.css [hunk 4]
```diff
@@ -2496,7 +2496,7 @@ ul#menu ul.ssmenu {
 }
 
 .copyright {
-   font-size: 9px;
+   font-size: 9px!important;
```
Comment (orthagh) [inline css/styles.css]: Could you explain this important ?

Comment (trasher) [inline css/styles.css]: "a, a:link" will have precedence; it was not required before because class was applied on a span element.

Comment (orthagh) [inline css/styles.css]: it certainly better to use a more specific rule than "important!" declaration.

```css
a.copyright {

}

Comment (trasher) [inline css/styles.css]: Fixed

**Comment by orthagh** [comment]
ok for me. I merge

**Pull request merged**